test/functional/deprecate_warning.test.js

```diff
@@ -22,68 +22,59 @@ describe('Deprecation Warnings - functional', function () {
   });
 
   const defaultMessage = ' is deprecated and will be removed in a later version.';
-  it.skip('node --no-deprecation flag should suppress all deprecation warnings', {
-    metadata: { requires: { node: '>=6.0.0' } },
-    test: function (done) {
-      exec(
-        'node --no-deprecation ./test/tools/deprecate_warning_test_program.js',
-        (err, stdout, stderr) => {
-          expect(err).to.not.exist;
-          expect(stdout).to.be.empty;
-          expect(stderr).to.be.empty;
-          done();
-        }
-      );
-    }
+  it.skip('node --no-deprecation flag should suppress all deprecation warnings', function (done) {
```

Comment:
I know these were skipped already but it seems we don't have any ticket to track unskipping them. It might be okay for us to drop these, we don't need to test that nodejs correctly implements the `--no-deprecation` flag. This test may have been more relevant when we had a wider array of node versions to test.